nist/int.go

```diff
@@ -416,3 +434,14 @@ func (i *Int) HideDecode(buf []byte) {
 	i.V.SetBytes(buf)
 	i.V.Mod(&i.V, i.M)
 }
+
+// SetEndianess will set the endianness of the value to end.
+func (i *Int) SetEndianness(end binary.ByteOrder) {
```

Comment:
Fine with me.  No real need for getters/setters if there's no important internal abstraction to be protected.